lib/ide/ast/impl/src/lib.rs

```diff
@@ -225,7 +219,7 @@ impl Serialize for Ast {
         if self.id.is_some() {
             state.serialize_field(ID, &self.id)?;
         }
-        state.serialize_field(SPAN,  &self.span)?;
+        state.serialize_field(SPAN, &self.size)?;
```

Comment:
This and some other places in code are inconsistent. Now we have "span" and "size" poinitng to the same thing/. Moreover, I dont like the word "size" - it could be memory size or something else. Span is more unambiguaous.

Reply:
Span collides with concept `(position + length)` which seems to be the default meaning of span (and so it was in Scala codebase).
I much prefer `size` or `length`.